gatherers/manifest.js

```diff
@@ -25,7 +25,7 @@ class Manifest extends Gather {
 
   static _loadFromURL(options, manifestURL) {
     if (typeof window !== 'undefined') {
-      const finalURL = (new window.URL(options.driver.url).origin) + manifestURL;
+      const finalURL = (new window.URL(options.driver.url).origin) + '/' + manifestURL;
```

Comment:
testing against `window` might get us in trouble if we aren't careful with how we stub out the environment for running devtools frontent code